test/requireHelper.js

```diff
@@ -0,0 +1,13 @@
+window.bkg = {
+    settings: require('../shared/js/settings'),
+    ATB: require('../shared/js/atb'),
+    Companies: require('../shared/js/companies'),
+    tabManager: require('../shared/js/tabManager'),
+    https: require('../shared/js/https'),
+    trackers: require('../shared/js/trackers'),
+    utils: require('../shared/js/utils'),
+    surrogates: require('../shared/js/surrogates'),
+    Site: require('../shared/js/site'),
+    abp: require('abp-filter-parser'),
+    tosdr: require('../shared/data/tosdr')
+}
```

Comment:
You can't `require` files outside of the Browserify bundle normally, so I'm using this as a shim to expose them to the global scope. This gets bundled up in special background.js file.